stix_shifter/stix_transmission/src/modules/utils/RestApiClient.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, host, port=None, cert=None, headers={}, url_modifier_function
         self.url_modifier_function = url_modifier_function
 
     # This method is used to set up an HTTP request and send it to the server
-    def call_api(self, endpoint, method, headers=None, params=[], data=None, urldata=None):
+    def call_api(self, endpoint, method, headers=None, params=[], data=None, urldata=None, timeout=None):
```

Comment:
but here, the default value is not equal the default value constant, it is puzzling.

Reply:
I think the idea was if any module's api_client didn't pass in the default value, the timeout would be set to None.

Reply:
it is also an option. In that case 'DEFAULT_PING_TIMEOUT_IN_SECONDS' has to be renamed and to become module local to modules where it is used.